### PR TITLE
Update DG

### PR DESCRIPTION
**Pull request by tonishka**

### CODE REVIEW

File: docs/diagrams/plantuml/Find.puml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@startuml
+'https://plantuml.com/activity-diagram-beta
+
+start
+:User executes find command;
+if (is Field given) then (true)
```
Comment (ckcherry23) [inline docs/diagrams/plantuml/Find.puml:6]: I think there is an error in the branch node notation.

Comment (tonishka) [reply docs/diagrams/plantuml/Find.puml:6]: Thanks for pointing it out, I'll rectify it!

File: docs/diagrams/plantuml/Find.puml [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 start
 :User executes find command;
-if (is Field given) then (true)
+if () then ([is Field given])
```
[the remark this answers is not in the document]
Comment (tonishka) [reply docs/diagrams/plantuml/Find.puml:6]: Sounds like a good idea!

[the remark this answers is not in the document]
Comment (NicsunXnus) [reply docs/diagrams/plantuml/Find.puml]: The image of the activity diagram shown in the file changes section does not seem to display the new changes. But other than that, lgtm

**Review by ckcherry23 — APPROVED** [review]
LGTM

**Pull request merged**